src/units/abilities.cpp

```diff
@@ -2028,7 +2028,7 @@ effect::effect(const unit_ability_list& list, int def, const_attack_ptr att, EFF
 			}
 		}
 
-		if(wham == EFFECT_CLAMP_MIN_MAX){
+		if(wham == EFFECT_CLAMP_MIN_MAX || wham == EFFECT_CUMULABLE){
```

Comment:
Why is this adding `EFFECT_CUMULABLE` when the three other C++ code changes are passing in `EFFECT_CLAMP_MIN_MAX`?

Reply:
For what min_value and max_value  usable in leadership where cumulable is used